app/src/components/governance/ProposalStatusBar.vue

```diff
@@ -26,13 +26,23 @@
         </div>
         <div>ID: {{ proposal.proposalId }}</div>
       </div>
-      <div class="vote-data">
-        <span>{{ votePercentage | percentInt }} of {{ stakingDenom }}</span>
-        <span v-if="voteCount">({{ voteCount }} Votes)</span>
+      <div class="vote-data-container">
+        <div class="vote-data">
+          <span>{{ votePercentage | percentInt }} of {{ stakingDenom }}</span>
+          <span v-if="voteCount">({{ voteCount }} Votes)</span>
+        </div>
+        <div v-if="proposal.type === 'TREASURY'" class="vote-data">
+          <span>Threshold:</span>
+          <span>{{ proposal.detailedVotes.votingThresholdYes }} Votes</span>
+        </div>
       </div>
       <ProgressBar
         size="large"
-        :val="votePercentage * 100"
+        :val="
+          proposal.type === 'TREASURY'
+            ? proposal.detailedVotes.votingThresholdYes
```

Comment:
this shows the threshold not the current status am I wrong? Shouldn't it be: yesVotes / votingThresholdYes?

Reply:
We could actually display both. But the latter is already visually displayed on the progress bar. But we could this display it as a percentage too :shrug: Not a bad idea

Reply:
let's have a call on this

Reply:
Tuesday?

Reply:
oops, we forgot about this, didn't we?